website/docs/guides/container_service_extension_install.html.markdown

```diff
@@ -1,79 +1,131 @@
+  used in the sample configuration, that can be rendered correctly with the Terraform built-in function `templatefile`.
+  (Note: In `terraform.tfvars.example` the path for the CSE v4.1 RDE contents is already provided).
+- `capvcd_version`: The version for CAPVCD. The default value is **"1.1.0"** for CSE v4.1.
+  (Note: Do not confuse with the version of the `capvcdCluster` [RDE Type][rde_type],
+  which **must be "1.2.0"** for CSE v4.1 and cannot be changed through a variable).
+- `cpi_version`: The version for CPI (Cloud Provider Interface). The default value is **"1.4.0"** for CSE v4.1.
+- `csi_version`: The version for CSI (Cloud Storage Interface). The default value is **"1.4.0"** for CSE v4.1.
+- `github_personal_access_token`: Create this one [here](https://github.com/settings/tokens),
+  this will avoid installation errors caused by GitHub rate limiting, as the TKGm cluster creation process requires downloading
+  some Kubernetes components from GitHub.
+  The token should have the `public_repo` scope for classic tokens and `Public Repositories` for fine-grained tokens.
+- `bootstrap_vm_sizing_policy`: The [VM Sizing Policy][sizing] used by the Bootstrap VM.
+- `http_proxy`: Address of your HTTP proxy server. Optional in the sample configuration.
+- `https_proxy`: Address of your HTTPS proxy server. Optional in the sample configuration.
+- `no_proxy`: A list of comma-separated domains without spaces that indicate the targets that must **not** go through the configured proxy. Optional in the sample configuration.
+- `syslog_host`: Domain where to send the system logs. Optional in the sample configuration.
+- `syslog_port`: Port where to send the system logs. Optional in the sample configuration.
+- `node_startup_timeout`: A node will be considered unhealthy and remediated if joining the cluster takes longer than this timeout (seconds, defaults to 900 in the sample configuration).
+- `node_not_ready_timeout`: A newly joined node will be considered unhealthy and remediated if it cannot host workloads for longer than this timeout (seconds, defaults to 300 in the sample configuration).
+- `node_unknown_timeout`: A healthy node will be considered unhealthy and remediated if it is unreachable for longer than this timeout (seconds, defaults to 300 in the sample configuration).
+- `max_unhealthy_node_percentage`: Remediation will be suspended when the number of unhealthy nodes exceeds this percentage.
+  (100% means that unhealthy nodes will always be remediated, while 0% means that unhealthy nodes will never be remediated). Defaults to 100 in the sample configuration.
+- `container_registry_url`: URL from where TKG clusters will fetch container images, useful for VCD appliances that are completely isolated from Internet. Defaults to "projects.registry.vmware.com" in the sample configuration.
+- `bootstrap_vm_certificates`: Certificate(s) to allow the ephemeral VM (created during cluster creation) to authenticate with.
```

Comment:
Is how CSE describes the VM that creates the cluster, also referred as "Bootstrap VM". If I remember correctly, even the VM is called that way when a new cluster is requested (`EPHEMERAL_VM`). In this case, I copied the exact descriptions from UI to avoid confusing users with my own words.